.github/workflows/black.yaml

```diff
@@ -0,0 +1,29 @@
+name: Black Formatting
+
+on: [pull_request, push]  # NOLINT
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2  # NOLINT
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install -r requirements.txt
+        pip3 install black
```

Comment:
I would change all of these to use `python -m pip ...`. Since the environment is already configured to use a specific Python version it will use the correct version of pip for these tests. More guidance at https://snarky.ca/why-you-should-use-python-m-pip/.